src/hotspot/share/gc/shared/gcConfig.cpp

```diff
@@ -96,18 +96,41 @@ void GCConfig::fail_if_non_included_gc_is_selected() {
 }
 
 void GCConfig::select_gc_ergonomically() {
-  if (os::is_server_class_machine()) {
+  if (strcmp(ErgonomicsProfile, "shared") == 0) {
+    if (os::is_server_class_machine()) {
 #if INCLUDE_G1GC
-    FLAG_SET_ERGO_IF_DEFAULT(UseG1GC, true);
+      FLAG_SET_ERGO_IF_DEFAULT(UseG1GC, true);
```

Comment:
Generally we don't touch whitespace

Reply:
This one is because we added a new level in the if block. See line 99.